helper.py

```diff
@@ -14,9 +14,17 @@ def convertSymbol(imgAlt):
     Coverts a card symbol image to its corresponding text -
     Black = B, Blue = U, Red = R, Green = G, White = W, Variable Colorless = X, Colorless = C, Phyrexian Red = P, Tap = Tap, 
     E = Energy
-    '''   
-    if (imgAlt == "Variable Colorless"):
-        return('X')
+    '''
+    imgAltWords = imgAlt.strip().split()
+    if (len(imgAltWords) > 1):
```

Comment:
Ran into a weird issue where "Half a White" was there and is more than 1 word and isn't being parsed.
Also, this whole thing assumes that when we have a multi mana cost, it's ok to have 2 letters with a '|' in-between, before we only had 1 value to represent 1 discrete mana cost.

Reply:
How is that even a thing in Magic, can you even tap Half a mana? Does it mean you could cast two for 1 mana tap?

Reply:
Good catch, that's super weird. Cards from the Unhinged and Mystery Booster sets introduce a few edge cases, and looks like that's one of them. Those cards aren't tournament-legal, so I've been debating whether to remove them or not